### PR TITLE
show query on list page for smart groups

### PR DESCRIPTION
**Pull request by nicpottier**
fixes: https://github.com/rapidpro/rapidpro/issues/1343

Screen Shot 2020-12-18 at 11 31 11 AM

Note these do get a bit weird when you do additional querying.. 

Screen Shot 2020-12-18 at 11 32 14 AM

### CODE REVIEW

File: templates/contacts/contact_filter.haml [hunk 1]
```diff
@@ -4,8 +4,10 @@
 
 -block title
   {{current_group.name}}
+
 -block subtitle
   -if current_group.is_dynamic
+    .font-mono.text-sm
+      {{ current_group.query }}
     .hidden.lbl-secondary.linked{onclick:"goto(event)", href:"{% url 'contacts.contact_filter' current_group.uuid %}"}
```
Comment (nicpottier) [inline templates/contacts/contact_filter.haml:12]: @ericnewcomer is this hidden div something?

Comment (rowanseymour) [reply templates/contacts/contact_filter.haml:12]: definitely looks like a bit a little bit of cruft @ericnewcomer ?

Comment (ericnewcomer) [reply templates/contacts/contact_filter.haml:12]: I'm not sure I follow, is what a hidden div?

**Comment by ericnewcomer** [comment]
We are pretty aggressive about letting them know their search was unsuccessful (also the main treatment really ought not be a card). Would be great if it just stopped here..
Screen Shot 2021-01-04 at 9 44 56 AM

File: templates/contacts/contact_filter.haml (hunk 1 above)
Comment (nicpottier) [reply templates/contacts/contact_filter.haml:12]: There's a `.hidden` div here which ya, doesn't show.. is it meant to or is that cruft left over from redesign?

Comment (ericnewcomer) [reply templates/contacts/contact_filter.haml:12]: Ah! I see, was looking at the wrong line. Yes, that is design cruft!

**Review by ericnewcomer — APPROVED** [review]

**Comment by rowanseymour** [comment]
@nicpottier wanna pull out that hidden div before we merge this?

**Comment by nicpottier** [comment]
Sure. Updated.

**Pull request merged**